Stratis.Bitcoin/Logging/LogsExtension.cs

```diff
@@ -7,99 +7,181 @@
 using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.Configuration.Settings;
 using Stratis.Bitcoin.Utilities;
+using NLog;
+using NLog.Config;
+using NLog.Targets;
+using System.Text;
 
 namespace Stratis.Bitcoin.Logging
 {
+    /// <summary>
+    /// Integration of NLog with Microsoft.Extensions.Logging interfaces.
+    /// </summary>
     public static class LogsExtension
     {
-        public static void AddFilters(this ILoggerFactory loggerFactory, LogSettings settings)
+        /// <summary>Width of a column for pretty console/log outputs.</summary>
+        public const int ColumnLength = 16;
+
+        /// <summary>Currently used node's log settings.</summary>
+        private static LogSettings logSettings;
```

Comment:
I think we should avoid static fields for logging, now that we have a DI all logs should be inject-able.
Is there any place this is used?